### PR TITLE
Finally Added Search Feature

### PR DESCRIPTION
**Pull request by s-ankur**
Sorry for the messy code, Will reorganize soon
Solved Issue #1

### CODE REVIEW

**Comment by piyush27ranjan** [comment]
Bro you have sent 2 PRs from 2 different branch which is going to have merge conflicts can create one PR for both

**Comment by s-ankur** [comment]
This is the only one!

**Comment by piyush27ranjan** [comment]
You both have made changes in main.py and after merging one there will be merge conflicts. can you two work on that together. @Soumyajitdey

File: main.py [hunk 1]
```diff
@@ -15,8 +15,13 @@
 from PIL import ImageTk, Image
 import eyed3
 
+    
 
 def tkinter_window(location, audiofile):
+    inputs={}
+    def get_entry():
```
Comment (piyush27ranjan) [inline main.py]: you have defined a function inside a function. Can you define it outside, it seems a bit funky

**Comment by Soumyajitdey** [comment]
Ok you can decline my pr. and after merging #31  i will again sent my pr. Is that will be better or not?

**Comment by s-ankur** [comment]
Thanks!

File: main.py (hunk 1 above)
Comment (s-ankur) [reply main.py]: Ill try to make it prettier but ultimately there needs to be a nested function for the callback

**Pull request merged**